src/main/scala/edu/berkeley/ce/rockslicing/NumericUtils.scala

```diff
@@ -5,7 +5,7 @@ import scala.math.BigDecimal
  * Numerical utilities for implementation and test cases.
  */
 object NumericUtils {
-  val EPSILON = 1.0e-6
+  val EPSILON = 1.0e-5
```

Comment:
Okay, I'll implement it in `approximateEquals`. This way it allows us to at least keep track of where other things might be losing accuracy. Perhaps once we get round to looking at floating point arithmetic in our code, this tolerance should get tightened back up.